### PR TITLE
Hcephes

### PR DESCRIPTION
**Pull request by Horta**

### CODE REVIEW

File: recipes/hcephes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  url: https://github.com/limix/hcephes/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win32 or linux32 or (win64 and py27)]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py36]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please add...

```yaml
    - vc14  # [win and py35]
```

File: recipes/hcephes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/limix/hcephes/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win32 or linux32 or (win64 and py27)]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please add `or (win and py35)`.

File: recipes/hcephes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - python  # [win]
+    - toolchain
+    - cmake
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py36]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please add...

```yaml
      - vc 14  # [win and py35]
```

File: recipes/hcephes/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  run:
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py36]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please add...

```yaml
      - vc 14  # [win and py35]
```

File: recipes/hcephes/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+test:
+  commands:
+    - test -e $PREFIX/lib/libhcephes.so  # [linux]
+    - test -e $PREFIX/lib/libhcephes_static.a  # [linux]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please change `linux` to `unix`.

File: recipes/hcephes/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    - test -e $PREFIX/lib/libhcephes.dylib  # [osx]
+    - test -e $PREFIX/lib/libhcephes_static.a  # [osx]
```
Comment (jakirkham) [inline recipes/hcephes/meta.yaml]: Please drop this line.

Comment (Horta) [reply recipes/hcephes/meta.yaml]: This one? `- test -e $PREFIX/lib/libhcephes_static.a  # [osx]`  ? but that file is created by osx...

**Pull request merged**